plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -4413,6 +4448,25 @@ else if (mechanism.requireObject(EntityTag.class)) {
                 spawner.update();
             }
         });
+
+        // <--[mechanism]
+        // @object LocationTag
+        // @name last_interacted_slot
+        // @input ElementTag(Number)
+        // @description
+        // Sets the last interacted slot of a Chiseled Bookshelf inventory.
+        // @tags
+        // <LocationTag.last_interacted_slot
+        // -->
+        tagProcessor.registerMechanism("last_interacted_slot", false, (object, mechanism) -> {
+            if (!(object.getBlockState() instanceof ChiseledBookshelf)) {
+                mechanism.echoError("Mechanism 'LocationTag.last_interacted_slot' is only valid for Chiseled Bookshelves.");
+                return;
+            }
+            if (mechanism.requireInteger()) {
+                ((ChiseledBookshelf) object.getBlockState()).setLastInteractedSlot(mechanism.getValue().asInt() + 1);
```

Comment:
Shouldn't this be `- 1`? you add 1 when the value goes into Denizen, and substract 1 when it goes back into Minecraft